src/test/resources/org/cirdles/topsoil/chart/WetherillTest.js

```diff
@@ -0,0 +1,33 @@
+/* 
+ * Copyright 2014 CIRDLES.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buster.spec.expose();
+
+describe("Wetherill", function() {
+   before(function() {
+        var LAMBDA_235 = 9.8485e-10,
+            LAMBDA_238 = 1.55125e-10,
+            vX = Math.exp(LAMBDA_235 * 10) - 1,
+            vY = Math.exp(LAMBDA_238 * 10) - 1;
+        this.vector = new Vector2D(vX, vY);
+   });
+   
+   it("can instantiate an instance (non-prime)", function() {
+       var curve = wetherill(10);
+       buster.assert.equals(this.vector, curve);
```

Comment:
`t = 0` is a great point to add. When comparing floating points, you should also build in a tolerance. Buster should provide something for that.